app/views/journals/_journal_block.html.slim

```diff
@@ -6,7 +6,7 @@
 #journalBlock.collapse
   .row
     .col-xs-12
-      = button_link t('add_journal_entry'), { controller: 'journals', action: 'new', "#{controller_name.singularize}_id": journaled_for.id }, 'success'
+      = button_link t('add_journal_entry'), url_for([journaled_for, Journal]) + '/new', 'success'
```

Comment:
I think `url_for([journaled_for, Journal], action: :new)` should work here as well.

Reply:
I tryed that and for some reason I got a `wrong number of arguments (given 2, expected 0..1)` on that. Thats why I ended up like this.